### DISCUSSION TITLE
A version upgrade issue

### DISCUSSION THREAD

**Discussion by superflq** [winpython/winpython#1429] [Q&A]
The version I am currently using is WinPython64-3.11.8.0=Python 3.11.8 64 bit with PyQt5+Spyder+Torch,
The latest released WinPython64-3.11.9.0dot=Python 3.11.9 64 bit only without PyQt5+Spyder+Torch and other libraries.
Can you please advise on how to upgrade to version 3.11.9 on the existing basis?
thank you.

**Comment by stonebig** [comment]
as an experiment, you may try: 
- copy your WPy64-31180\python-3.11.8.amd64 into a WPy64-31190frankenstein
- copy (overwrite what was existing, leave what's not replaced) the containt of ;
   - your "dot" WPy64-31190\python-3.11.9.amd64
   - into  WPy64-31190frankenstein\python-3.11.8.amd64
- try

maybe a (command line dos): 
`xcopy /Y WPy64-31190\python-3.11.9.amd64 WPy64-31190frankenstein\python-3.11.8.amd64 `

**Comment by superflq** [comment]
ok, i try it
